powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationTemplateDeleteRequest.java

```diff
@@ -28,6 +30,8 @@
 @Data
 public class OperationTemplateDeleteRequest {
 
+    @Schema(description = "Operation template identifier")
+    @NotNull(message = "Operation template identifier must not be empty when deleting operation template")
```

Comment:
Constraint reference
```suggestion
    @NotNull(message = "Operation template identifier must not be null when deleting operation template")
```